plugins/catalog-react/src/alpha/blueprints/EntityCardBlueprint.ts

```diff
@@ -35,25 +37,30 @@ export const EntityCardBlueprint = createExtensionBlueprint({
     coreExtensionData.reactElement,
     entityFilterFunctionDataRef.optional(),
     entityFilterExpressionDataRef.optional(),
+    entityCardAreaDataRef.optional(),
   ],
   dataRefs: {
     filterFunction: entityFilterFunctionDataRef,
     filterExpression: entityFilterExpressionDataRef,
+    area: entityCardAreaDataRef,
```

Comment:
@Rugvip I could not name it variant because it conflicts with cards that already expect that prop:
![Uploading card variant name conflict.png…]()

Reply:
Followup in https://github.com/backstage/backstage/pull/28930